circle.yml

```diff
@@ -1,20 +1,113 @@
-dependencies:
-  override:
-    - curl -L https://atom.io/download/deb -o atom-amd64.deb
-    - sudo dpkg --install atom-amd64.deb || true
-    - sudo apt-get update
-    - sudo apt-get -f install
-    - node --version
-    - npm --version
-    - atom --version
-    - rm -rf node_modules; npm install
-    - npm run compile
+version: 2
 
-test:
-  override:
-    - npm run lint
-    - npm test
+defaults: &defaults
+  working_directory: /tmp/project
+  docker:
+    - image: arcanemagus/atom-docker-ci:stable
+  steps:
+    - run:
+        name: Update distro
+        command: |
+          sudo apt-get update && \
+          sudo apt-get -f install --assume-yes --quiet --no-install-suggests \
+            --no-install-recommends
+    - run:
+        name: Create VFB for Atom to run in
+        command: /usr/local/bin/xvfb_start
+    - run:
+        name: Cleaning package
+        command: ${APM_SCRIPT_PATH} clean
```

Comment:
This "Cleaning package" step should be after the "Package dependencies" (`${APM_SCRIPT_PATH} install`) step.